bulk-model-sync-gradle/src/main/kotlin/org/modelix/model/sync/bulk/gradle/config/ModelSyncGradleSettings.kt

```diff
@@ -43,6 +43,7 @@ data class SyncDirection(
     internal val registeredLanguages: Set<ILanguage> = mutableSetOf(),
     internal val includedModulePrefixes: Set<String> = mutableSetOf(),
     internal var mpsDebugEnabled: Boolean = false,
+    var continueOnError: Boolean = false,
```

Comment:
I wonder, if we should expose this only as a method like `enableContinueOnError()` for consistency as sync direction only used methods for its settings up until now.

Reply:
I had that in my Gradle script and it looked a bit weird to me for just a single boolean.

Reply:
Updated to that pattern

Reply:
The name in the docs isn't updated. We could also get rid of the parameter as `enableContinueOnError(false)` wouldn't do anything.

Reply:
without the flag, that would be a one-way street. I added the flag so that one is also able to disable this, too.

I'll update the docs